### PR TITLE
feat(ui): Update transaction summary UI for trace view

### PR DESCRIPTION
**Pull request by Zylphrex**
Transaction summary is getting a revamp as part of the upcoming trace view designs. This change introduces some of these changes behind the `trace-view-quick` flag.

This change removes the sidebar and moves the event meta above the span view and the JSON preview is in the header actions. Upcoming changes to this page include

- ~Moving the download as JSON button to the header~
- Moving the operation breakdown to the left of the minimap.

# Screenshots

## Before

image

## After

image

### CODE REVIEW

File: src/sentry/static/sentry/app/views/performance/transactionDetails/content.tsx [hunk 1]
```diff
@@ -112,6 +114,8 @@ class EventDetailsContent extends AsyncComponent<Props, State> {
     const transactionName = event.title;
     const query = decodeScalar(location.query.query, '');
 
+    const hasQuickTraceView = organization.features.includes('trace-view-quick');
```
Comment (wmak) [inline src/sentry/static/sentry/app/views/performance/transactionDetails/content.tsx:117]: Should this be both `trace-view-quick` and `trace-view-summary`

**Review by wmak — COMMENTED** [review]
Should we do these changes in `app/views/eventsV2/eventDetails/content.tsx` too? (Took me a bit to realize this was only changing event details for Performance.

File: src/sentry/static/sentry/app/views/performance/transactionDetails/content.tsx (hunk 1 above)
Comment (Zylphrex) [reply src/sentry/static/sentry/app/views/performance/transactionDetails/content.tsx:117]: Hm... that's a good point, without `trace-view-summary`, `trace-view-quick` wouldn't work anyways. I'll add it.

**Comment by dashed** [comment]
@wmak Changing this for only transaction events in Discover might be inconsistent since error events won't have any special treatment. Might be worth getting feedback on this layout revision in Performance first before making a commitment to it in other areas of the product.

**Comment by wmak** [comment]
We want to show the quick-trace for error events too when they contain a trace, wouldn't that be keeping these two consistent?

**Comment by Zylphrex** [comment]
> We want to show the quick-trace for error events too when they contain a trace, wouldn't that be keeping these two consistent?



> We want to show the quick-trace for error events too when they contain a trace, wouldn't that be keeping these two consistent?

I think it makes more sense to leave it off of discover for now as that contains both errors and transactions. For simplicity, that can come later.

**Review by wmak — APPROVED** [review]
LGTM

**Pull request merged**